### DISCUSSION TITLE
Is it possible to get an email only response to an issue?

### DISCUSSION THREAD

**Discussion by manav-sharma69** [community/community#130969] [Projects and Issues] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I raised an [issue on axe-core's extension](https://github.com/dequelabs/axe-core/issues/4473#issuecomment-2192452416). After some time, I received an email response, but when I clicked the link "view it on GitHub", I did not find any comments there. Even if I visit the issue from GitHub directly, it does not show up.
So, is it possible to get an email only response to an issue and not see the response as a comment in issue on GitHub itself?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by davevad93** [comment, marked as answer]
Hi @manav-sharma69 ,

When you receive an email notification of a reply to an event but doesn't show up on GitHub, it's because has been deleted. Sometimes it happens to me in discussions too, I get an email notification, but then when I enter the discussion, the reply isn't there, which is because it has been deleted by the same user who made it.

**Reply by manav-sharma69** [reply]
Thanks a lot for response!
